index.html

```diff
@@ -4,11 +4,199 @@
+                <span class="remove h-3 w-3 cursor-pointer">
+                  <img
+                    class="absolute right-3 h-4 w-4"
+                    alt="cross-icon"
+                    title="cross-icon-image"
+                    src="images/icon-cross.svg"
+                  />
+                </span>
+              </label>
+            </li>
+            <li
+              class="list-item size-full h-14 list-none justify-between border border-blue-200 bg-blue-0 p-4 text-base text-blue-200"
+            >
+              <label class="list relative flex cursor-pointer items-center">
+                <input
+                  class="checked:bg-gradient-to-r peer mr-2.5 h-5 w-5 cursor-pointer appearance-none rounded-full border border-blue-200 from-blue-700 to-blue-800 checked:border-none checked:bg-blue-800 checked:bg-icon-checked checked:bg-center checked:bg-no-repeat"
+                  type="checkbox"
+                />
+                <span class="text-blue-400 peer-checked:text-blue-500 peer-checked:line-through"
+                  >Complete Todo App</span
+                >
+                <span class="remove h-3 w-3 cursor-pointer">
+                  <img
+                    class="absolute right-3 h-4 w-4"
+                    alt="cross-icon"
+                    title="cross-icon-image"
+                    src="images/icon-cross.svg"
+                  />
+                </span>
+              </label>
+            </li>
+          </ul>
+        </div>
+
+        <div
+          class="mob flex size-full h-14 justify-between border border-blue-200 bg-blue-0 p-4 text-base text-blue-500"
+        >
+          <div class="mob-count pt-1"><span>5</span> items left</div>
+          <div class="mob-clear cursor-pointer">Clear Completed</div>
+        </div>
+
+        <div
+          class="footer mt-5 flex size-full h-12 justify-between rounded-lg bg-blue-0 p-4 text-base text-blue-500"
+        >
+          <div class="filters m-auto flex">
+            <label class="filters-list cursor-pointer hover:text-blue-400">
+              <input class="filters-input peer opacity-0" type="radio" name="filter" checked="" />
+              <span class="peer-checked:text-blue-600">All</span>
+            </label>
+            <label class="filters-list cursor-pointer hover:text-blue-400">
+              <input class="filters-input peer opacity-0" type="radio" name="filter" />
+              <span class="peer-checked:text-blue-600">Active</span>
+            </label>
+            <label class="filters-list cursor-pointer hover:text-blue-400">
+              <input class="filters-input peer opacity-0" type="radio" name="filter" />
+              <span class="peer-checked:text-blue-600">Completed</span>
+            </label>
+          </div>
+        </div>
+        <div class="dragdrop fixed bottom-5 self-center text-base text-blue-300">
+          <p>Drag and drop to reorder list</p>
+        </div>
+      </div>
```

Comment:
![image](https://github.com/moshoodabdullahi/todo-list-app/assets/105943931/d450ec22-baea-4582-a278-9ac6c036fca9)
here is what you footer looks like please lets fix this